Automattic/simplenote-macos
# Add finalize release lane

This PR adds a lane that automates some steps of the release process.
The PR builds on top of https://github.com/Automattic/simplenote-macos/pull/352 and should be merged after that one.

# To Test
1. Checkout a new `release/1.6` branch and push it with `-u`
2. Change the versions in `config/Version.public.xcconfig` to `1.6` and `1.6.0.0`.
3. Commit the changes. 
4. Add a `1.6.0.0` tag and push it.
5. Run `bundle exec fastlane finalize_release`. 
6. When prompted, verify that the code picked the right version and confirm.
7. Verify that the script performs all the checks without errors. 
8. Verify that the `1.6.0.0` tag has been deleted from the local repository and from GitHub.
9. Verify that a new `1.6` has been added. 

### Clean up
10. Delete the `1.6` tag from your local repository and from GitHub.
11. Delete the `release/1.6` branch from your local repository and from GitHub.
